install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -242,8 +242,25 @@ spec:
                   docker kill "${container_id}" || true
                 done
               else
-                # COS-beta (with containerd). Some versions of COS have crictl in /home/kubernetes/bin.
-                for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do PATH="${PATH}:/home/kubernetes/bin" crictl stopp "$(sed 's/\r//;1q' $f)" || true; done
+                if [ -d /var/lib/cni/networks/ ]; then
+                    echo "Sleeping for 60 seconds..."
+                    sleep 60
```

Comment:
Why is the `sleep` necessary?

Reply:
Would also like to know the reason (it's not documented in the code). Also, we can't sleep here, this would delay the start of the agent Pod by 1 minute..

Reply:
Ok I will delete the sleep.